modules/openshift-cluster-maximums.adoc

```diff
@@ -25,7 +25,7 @@
 | 250
 | 250
 | 250
-| 250
+| 500 footnoteref:[podspernode, This was tested on a cluster with 100 worker nodes with 500 pods per worker node. The default `maxPods` is still 250. To get to 500 `maxPods`, the cluster must be created with a `hostPrefix` of `22` in the `install-config.yaml` file and `maxPods` set to `500` using a custom KubeletConfig.]
```

Comment:
Do we want to provide a link to the KubeletConfig section of the "Recommended host practices" topic?  I don't feel strongly about it.  Too minor of a link to be worth the hassle of maintaining it maybe.

Reply:
Right now due to tooling, we're limited in what we can link to. We can't link to a module from a module, but that may change soon. I'll omit the link for now, but we can consider adding it in the future. Thanks, Mike!